pkg/chain/celo/celoutil/gen/error_resolver.go

```diff
@@ -1,22 +1,18 @@
-package celoutil
+// Code generated - DO NOT EDIT.
+
+package gen
```

Comment:
I think this will be inconvenient in places where we'll also use the original `celo/celoutil` package. Also, that package structure looks weird and adds a redundant level to the package structure. Personally, I prefer to have everything in `celo/celoutil` package instead of trying to separate generated code at all costs by complicating the package structure and playing with import aliases in the imported code. After all, generated files are easily distinguishable because they have `Code generated - DO NOT EDIT.` comments at their top.

Reply:
`gen` is just a pattern that indicates we are dealing with generated code. I do agree import aliases could be unwieldy but at the same time, I do not think we should use `gen` as a package name. I need to think more about it, maybe there is a name under which we can aggregate all of them: `celo/ceolutil/gen/somename` 🤔